sb.web.app/src/router/index.js

```diff
@@ -59,6 +59,12 @@ const routes = [
     component: NewArticle,
     props: true
   },
+  {
+    path: '/edit/article/:artID',
+    name: 'Edit_Article',
+    component: NewArticle,
```

Comment:
Should we change the name of the component? EditArticle routed to by 'new article' makes sense to me but a component called 'NewArticle' as the way to **Edit** an article feels quite confusing.. 

But its no biggie... 

Of course idealyl we'd call it editItem but i can live with edit Article for consistency

Reply:
So what's happened here is I've re-used the NewArticle component in the editing workflow instead of creating a new one. @utunga